src/patternfly/components/Page/page.scss

```diff
@@ -427,8 +438,12 @@ $pf-c-page--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg"
 }
 
 .pf-c-page__sidebar-body {
-  padding-top: var(--pf-c-page__sidebar-body--PaddingTop);
-  padding-bottom: var(--pf-c-page__sidebar-body--PaddingBottom);
+  padding-right: var(--pf-c-page__sidebar-body--PaddingRight);
+  padding-left: var(--pf-c-page__sidebar-body--PaddingLeft);
+
+  &:last-child {
+    flex-grow: 1;
+  }
 
   &.pf-m-menu {
     --pf-c-page__sidebar-body--PaddingTop: var(--pf-c-page__sidebar-body--m-menu--PaddingTop);
```

Comment:
Doesn't look like this and the below PaddingBottom vars will do anything since they're not applied to a property.

Reply:
Push an update to remove these lines and the CSS vars that weren't being used.